air_pollution.ipynb

```diff
@@ -0,0 +1,1292 @@
G3zBi4vRSVIcoJrOCqgOnIRLMFti1DVoo6wgGXVpANQ5hja8rg9gegTdIaWkpJrOFJ1YcmYAdbdaTag8jNdJIWqSRVLuRFLsx0KGiKAq9evUiIyODvXv3snLlSr799tuAkOrbt2+XE1Ip6WHYI3Ws+bGWn5bVMWDoOWQ8NhW+nU/9t/MRK5egTD4f5eKrUI5df2kFwl2H+O9riOU5kDkA9ca7m1ko+LyCbRtd5O+uJ9yuMm6KrZlX0YnwaYJVB2uYv72c7SUurGEqlw2I5qJ+UcRYzsyZXBJaKIqC2aJgtqjEJTS9zV13jJdVlUZpsZcD+5p2BlrDmxauh0fosIarZzT+JySx2CAsDBz+z0CzXg16DdQ3u/0Z4RmnYJwpdm5FfPOp/3No6JgW7+PzNZpe1lNV4TtqmS4Mi7VzlGMUFRWxadMmampqMBqNmEwmjEbjCf82Go1d4jvJ3/Gr4qwFMgf6DTVDhC4toOrrBZoPzFUHmwwRLi0tRW+LAg1uzUqgtt5HQaWbfRX1AffbRuKtBtIiw0i1GwPiKiU9g169egWE1DfffMOqVau6pJCKiNRx1rnhbFzlZNvGOspLDQy/4GriL59F2duvIJZ9jVjxPcq5M1HOn4lisrRqv+JAvn/JruiAf9TCJVc3GxRZ6fDPsaup0sjoE8aAYeZWf5G5vRrf7a3ksx3lFFZ76GEz8OtR8WT3jgyIYkn3wWjyu6XHxDf9yPN4/BmrJp2BDr+fVQAFrFY1UFtlC/d7WdnCdafduBBsFEXxd+JVlAN+M81gZqB8muCLXAdDelhanREWdU6/uW5sD5Sf/6rZ7Y3LdAV766l3d75lOk3T2L17Nxs2bKCoqAiDwUBMTAy1tbW43W7cbjc+34mtJ8LCwk4otDqL+LLaVJw1PpQ+AxGb1yCqK1HC7cEOq2sLqEYPKFPZPpT+/vonn89HeXk5hsR+6N1wdi87+qPS9j5NUFzrYV+Fm4JKNwUVbgoq6ll3qJZGXaUq0DM8jFS7idRB2fRyFlG6axPffPNNICPVp0/nnDPXEgaDwqgJFvJ2utm2sY5l31aTfVES6i9vRmTPQHz6LuLz9xFLvkS5+Er/r0F9y9kdIQRiyVeID/8FFivqnL+gDBjW7D57c93s2FyHIUwha4qV+ITWZYscLi9f5Dr4apeD6nqNvjEmrj0rjnHJ4ei6y/KMpNUYDApRMXqiYpp+FPq8gppqrZlRaHGht0lnoMmiHDEIPaoz8ExHB3UIUbGIxgxUkGugfiyoptTp5aYxrZ9VKj74F5SVoN77VxST2X9d4zLdLjdFB45apusTRkx851imq6urY8uWLYGMk91uZ/LkyQwYMACjsakru9frpa6uLiCoGv9u6Tq32015eXmbia8T/d3W330Wq0ppsReGNlgR7doGI8e36TFOh64toBosDMzVhRA7CICKigo0TcOhWEkKNzYRTwA6VaFneBg9w8MYlxIeuN7jExRW1zdkqtwN/9fx0/5qBGGgjiIhopg+NXv5+uuv+e6Hn0gfNJyRg/oTbwsL2aGzrUVRFHr1MxEZrWftilo+/98BhowykZKRiHLz7xF5O9E+ftu/JJfzGcpl16KMmtjEi0U4a/wjFtb+CINGoP7qDpSIpkt/LqfGhpVOSou9JCQZGDrGjLEVX0YFFW7m7yhncV4VPk2QlWJjZv9o+seZO8WHpiS00OkV7FE67FHNR9vU1hwzM7BSo6zE3aQzMMyoNGSr1ECNVXiEDqMpdArYlchoxO7tgD8D5XDVn2SL9uOzHeUkhhsYndQ6816xYSXih29Rpl+BkjkQn09wqKCevTsblukMCr36NizTneFEgo6ivLycjRs3sn37drxeL8nJyUydOpX09PTjChK9Xo/NZsNmO3XT4+OJr+P97XA4Apc7WnxZbTp8XqhPyMRgCPMXkksB1b4ETDTrygIWBiUlJQAc9FpIjWt9+7tBp5Aa6V/Gm5R25Hq3V+NgVX2DqIphnyODwsJ9xFTsZPeqJWxYu4YDtkws8amkRpr89VWRRlLtYUSbO8cvoqOJjtMz+bxwNq/xsGGVi/JSH4NHmtFl9EW961HYsg7t47cQrz2NSPsU9YrZKAOGIfbm+pfsKspQrpiNct5lzYzuDu2vZ9MaF5omGDbGTEpG2AmfHyEEmw47mbetnHWFtYTpFM7tbWdG/2gSI7revDpJ8FFVf8YpPEJHz6OuF0Lgqm1uFHqwoB7vUauBegMtZqzMVrXjPwuiYqGiHKFpQc1A7ShxsbOsjv8b3aNVPzRFdaX/h1hyBnXnXkXBZhf79jQs00WoDBllJjm9cyzTCSHYtWsXS5YsoaCgAJ1OR79+/Rg+fHiLNjttSSiKL4vFwvXXX99snFqgE8+tYs/oi9i17ZRjbg+6toCq1dApGgZPTcDCoKysDFVV2e8OY9IpDqlsCaNepVe0iV7RR6/bp1LjHseazbns2LgWW+VGvHV57C3rzXfE+u2WAWuYSpq9UVAZSY0MI81uJMIU2qfFaFI5b0YiPy45wK5tbirKfYye2DB7bsgo1EHDET8tQcz/D9pzf4Le/SF/F0TGoN7zOErv/k325/EItq5zsT+/nsho/xw7W/jxfzV6fIIf9lUxf0c5eQ43kSYdvxwaywV9o4g4TjelRNKeKIqCxabD0kJnoLuu+Wibw4c87M87shao6o7qDLQfEVdWWzt2BkbGgM8LNVWYDTqcQSoi/2xHOdYwlbN7nbymRQiB752XcRh6sm/S3RR96UII6JGkp1cfY6dZpvN4PGzfvp2NGzficDiwWCyMGzeOwYMHY7G0ro40mJyp+GpJaBUWFrJlyxYqKiqaicdGAVVboxHZdxDii48QLieKObjPVWh/U58hLqeGSXGhQKCIvLS0FKs9CqGopzzl+1SwGQ1MHT2YySMHsmvXLlatWoW+fD2jomNI7j+CWmsC+6vqKahws2xfFbX1Rz68Ik06f8H6MeKqceRCKKAe7V7+k5Nl39QwPMt
```

Comment:
On the xlabels I see some dates but not all and there's no time.
What is the most polluted area and why isn't it mentioned in the graph?